ui/src/lib/types.ts

```diff
@@ -95,7 +95,15 @@ export type PublicationCreationStoreType = {
     updateReferences: (references: Interfaces.Reference[]) => void;
 };
 
-export type JSONValue = string | number | boolean | { [x: string]: JSONValue } | Array<JSONValue> | null | undefined;
+export type JSONValue =
+    | string
+    | number
+    | boolean
+    | { [x: string]: JSONValue }
+    | Array<JSONValue>
+    | null
+    | undefined
+    | Interfaces.Reference[];
```

Comment:
Interfaces.Reference[] can be removed as it doesn't help in this case. We will find a better way for this